cooperator/models/company.py

```diff
@@ -100,21 +100,61 @@ def _compute_base_logo(self):
         translate=True,
         help="Text to display aside the checkbox to approve the generic rules.",
     )
+    cooperator_certificate_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Certificate email template",
+        domain="[('model', '=', 'res.partner')]",
+    )
+    cooperator_certificate_increase_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Certificate increase email template",
+        domain="[('model', '=', 'res.partner')]",
+    )
     send_certificate_email = fields.Boolean(
         string="Send certificate email", default=True
     )
+    cooperator_confirmation_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Share confirmation email template",
+        domain="[('model', '=', 'subscription.request')]",
+    )
+    cooperator_confirmation_company_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Company share confirmation email template",
+        domain="[('model', '=', 'subscription.request')]",
+    )
     send_confirmation_email = fields.Boolean(
         string="Send confirmation email", default=True
     )
+    cooperator_capital_release_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Capital release email template",
+        domain="[('model', '=', 'account.move')]",
+    )
     send_capital_release_email = fields.Boolean(
         string="Send Capital Release email", default=True
     )
+    cooperator_waiting_list_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Waiting list email template",
+        domain="[('model', '=', 'subscription.request')]",
+    )
     send_waiting_list_email = fields.Boolean(
         string="Send Waiting List email", default=True
     )
+    cooperator_share_transfer_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Share transfer email template",
+        domain="[('model', '=', 'res.partner')]",
+    )
     send_share_transfer_email = fields.Boolean(
         string="Send Share Transfer Email", default=True
     )
+    cooperator_share_update_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Share transfer email template",
```

Comment:
```suggestion
        string="Share update email template",
```

also, i think we decided to Capitalize Every Word of field descriptions.

Reply:
Regarding capitalisation: that wouldn't be consistent with the existing fields of this module. If we prefer internal consistency (which we should), we ought to change _all_ strings, which I'm going to consider out-of-scope.

Thanks for the typo though!